sdcm/nemesis.py

```diff
@@ -1533,9 +1533,10 @@ def choose_partitions_for_delete(self, partitions_amount, ks_cf, with_clustering
                 if exclude_partitions and partition_key in exclude_partitions:
                     continue
 
-                # The scylla_bench.test table is created WITH CLUSTERING ORDER BY (ck DESC).
-                # So first returned value is max cl value in the partition
-                cmd = f"select ck from {ks_cf} where pk={partition_key} limit 1"
+                # Get the max cl value in the partition according to the table's clustering order.
+                table_clustering_order = get_table_clustering_order(ks_cf=ks_cf, ck_name='ck', session=session)
+                reversed_order = 'desc' if table_clustering_order.lower() == 'asc' else 'asc'
```

Comment:
i think i understand here, but seems too confusing...
you need to run the query as `desc` if the table is set as `asc` to have reversed query...
maybe a small comment/doc here will help others understand it

Reply:
Added an additional comment about it.

Reply:
maybe it is only me, but the comment is still not 100% clear...